distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Upload/Operations/Delete.enso

```diff
@@ -167,6 +167,9 @@ type Source_DB_Table_Uploader
        to it.
        This method should be called after the invariant checks have passed -
        otherwise the upload itself could fail due to NULL constraint violations.
-    materialize self -> Table & DB_Table = case self of
-        Source_DB_Table_Uploader.Existing_DB_Query db_table -> db_table
-        Source_DB_Table_Uploader.Temporary_DB_Table recipe -> recipe.perform_upload
+    materialize self -> Table & DB_Table =
+        raw = case self of
+            Source_DB_Table_Uploader.Existing_DB_Query db_table -> db_table
+            Source_DB_Table_Uploader.Temporary_DB_Table recipe -> recipe.perform_upload
+        # explicitly uncover hidden types
```

Comment:
- a small contribution to #12292
- just writing `-> Table & DB_Table` does _not reveal hidden types_ from a value
- one has to _explicitly uncover them_ by `raw:(Table & DB_Table)`